src/test/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiServiceSpec.scala

```diff
@@ -54,38 +54,38 @@ class SubmissionApiServiceSpec extends ApiServiceSpec {
 
   private def createAndMonitorSubmission(wsName: WorkspaceName, methodConf: MethodConfiguration,
                                          submissionEntity: Entity, submissionExpression: Option[String],
-                                          services: TestApiService): SubmissionStatusResponse = {
```

Comment:
I don't think that I actually changed anything in this file, it's just weird indentation changes. I'll revert